### DISCUSSION TITLE
[Feat]: 希望添加差分宇宙的难度选择

### DISCUSSION THREAD

**Discussion by PikipikiTime** [moesnow/March7thAssistant#414] [Q&A] [answered]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我确定这是一个从未被提出和实现过的功能。
- [X] 我已经在 [Issue Tracker](https://github.com/moesnow/March7thAssistant/issues) 以及 [Pull Request](https://github.com/moesnow/March7thAssistant/pulls) 中找过我要提出的功能，没有找到相同功能的 Issue 和 PR。
- [X] 我已知晓并同意，此处仅用于建议新功能。若这个 Issue 是关于其他非主题的问题，则我的 Issue 可能会被无条件自动关闭/锁定。
- [X] 我已知晓并同意，如果我不按照模板提供的指示进行填写，则我的 Issue 可能会被无条件删除。

### 需要添加的功能内容

想要挂着刷差分的时候能选择难度

**Comment by shotamiao** [comment, marked as answer]
原版运行即可设置
